frontend/packages/dev-console/src/components/pipelines/pipeline-form/startPipelineModal.tsx

```diff
@@ -0,0 +1,75 @@
+import * as React from 'react';
+import * as _ from 'lodash';
+import { Formik, FormikValues } from 'formik';
+import {
+  createModalLauncher,
+  ModalComponentProps,
+} from '@console/internal/components/factory/modal';
+import { k8sCreate } from '@console/internal/module/k8s';
+import { PipelineRunModel } from '../../../models';
+import { Pipeline, PipelineResource, Param } from '../../../utils/pipeline-augment';
+import { StartPipelineForm } from './startPipelineForm';
+import { validationSchema } from './pipelineForm-validation-utils';
+
+export interface PipelineModalProps {
+  pipeline: Pipeline;
+  newPipelineRun: Function;
```

Comment:
```suggestion
  getNewPipelineRun: Function;
```
Also add type of the function.